CONTRIBUTING.md

```diff
@@ -207,19 +209,27 @@ Look inside a table
     ----+-----------+-----------
     (0 rows)
 
+### ER Diagram
+
+![ER Diagram](./files/ERDiagram.png)
+Beware, it might be outdated because it was generated manually
+(with [DBeaver](https://dbeaver.io)) and it's not automatically re-generated.
```

Comment:
Can you add a simple HOWTO? Did you install it from rpm or flatpack? How did you connect it to DB? What menu item did you hit to export it to PNG? This may sound trivial now, but from my experience, it will require a non-trivial amount of time to reproduce a few years later.

Reply:
It was trivial, I didn't even check any documentation, but I added a small HOWTO.